packages/hardhat-helpers/src/deploy.js

```diff
@@ -0,0 +1,20 @@
+export const deployContract = async (
```

Comment:
Do we also need a `deployUpgradableContract` helper?
Maybe in a follow up PR?

Reply:
yes, added too here